src/saveToCase.js

```diff
@@ -0,0 +1,475 @@
+define([
+    'vellum/form',
+    'jquery',
+    'underscore',
+    'vellum/mugs',
+    'vellum/parser',
+    'vellum/tree',
+    'vellum/util',
+    'vellum/widgets',
+    'tpl!vellum/templates/widget_update_case',
+    'vellum/core'
+], function (
+    form_,
+    $,
+    _,
+    mugs,
+    parser,
+    Tree,
+    util,
+    widgets,
+    widget_update_case
+){
+    function createsCase(mug) {
+        return mug ? mug.p.use_create : false;
+    }
+
+    function closesCase(mug) {
+        return mug ? mug.p.use_close : false;
+    }
+
+    function updatesCase(mug) {
+        return mug ? mug.p.use_update : false;
+    }
+
+    function addSetValue(mug) {
+        var path = mug.absolutePath;
+
+        if (createsCase(mug)) {
+            mug.form.addSetValue('xforms-ready', path + "/case/@case_id", mug.p.case_id);
+        }
+    }
+
+    var propertyWidget = function (mug, options) {
```

Comment:
minor: curious why you switched from `function funcName(arg)` to `var funcName = function (arg)` here? I always prefer the former as it seems like the idiomatic way to define a function. Granted, there are [scoping gotchas](http://javascriptissexy.com/javascript-variable-scope-and-hoisting-explained/) that you should understand, but as long as the function is defined before it is referenced and it's name does not collide with a variable name you should be fine.